BulkyBookWeb/Views/Order/Delete.cshtml

```diff
@@ -60,6 +63,75 @@
 			@Html.DisplayFor(model => model.City)
 		</dd>
 	</dl>
+	<table class="table table-bordered table-striped">
+		<thead>
+			<tr>
+				@if (User.IsInRole("Admin"))
+				{
+
+					<th>
+						OrderHeader Id
+					</th>
+				}
+				<th>
+					Product
+				</th>
+				<th>
+					Description
+				</th>
+				<th>
+					ISBN
+				</th>
+				<th>
+					Author
+				</th>
+				<th>
+					Image
+				</th>
+				<th>
+					Count
+				</th>
+				<th>
+					Price
+				</th>
+			</tr>
+		</thead>
+		<tbody>
+			@foreach (var obj in Model.OrderDetails)
+			{
+				<tr>
+					@if (User.IsInRole("Admin"))
```

Comment:
no need this, admin doesn't really know about order header

Reply:
remove the orderHeaderId